docs/about-core-concepts/extensions.rst

```diff
@@ -1,83 +1,132 @@
+ODC Geo
++++++++
 
+`odc-geo`_ provides low level geometry, CRS handling and reprojecting utilities, and the ``.odc``
+xarray extensions.
 
-.. _datacube-ows: https://github.com/opendatacube/datacube-ows
+ODC Loader
+++++++++++
+
+`odc-loader`_ Provides tools for loading data from files in local or cloud storage into
+xarray objects.
+
+(Depends on `odc-geo`_)
+
+ODC STAC
+++++++++
 
-Examples of this web server can be found in the `Australian National Map`_ service.
+`odc-stac`_ supports integrated data discovery from STAC API endpoints and loading of data into
+xarray objects.
 
-.. _`Australian National Map`: https://nationalmap.gov.au/#share=s-jfEZEOkxRXgNsAsHEC6xBddeS1b
+(Depends on `odc-geo`_ and `odc-loader`_)
+
+Datacube
+++++++++
+
+`datacube`_ (aka `datacube-core`_) supports integrated data discovery from a locally-maintained ODC
+database, and loading of data into xarray objects.  Also supports administrative collection-management
+activities.
+
+(Depends on `odc-geo`_.  Supports extended loading
+features using `odc-loader`_, but also has its own legacy loading engine)
+
+Publication Services
+--------------------
+
+Open Web Services
++++++++++++++++++
 
+`datacube-ows`_ provides WMS, WMTS, and WCS web service endpoints for serving data from one or more
+ODC databases, for supplying raw data or rendered visualisations to web map clients (like TerriaJS
+or Leaflet) or desktop GIS applications (like QGIS).  Includes a
+powerful configurable visualisation library.
+
+See `DEA Maps`_ or `Digital Earth Africa Maps`_ maps for examples of Datacube-OWS in action
+
+(Depends on `odc-geo`_ and `datacube-core`_)
 
 Data Cube Explorer
-------------------
+++++++++++++++++++
+
+`datacube-explorer`_ provides a web-based front end and a STAC-API endpoint for browsing and searching
+the contents of an ODC database.
 
-`Data Cube Explorer`_ is a web application for searching and browsing the metadata
-available from an Open Data Cube. It has rich visualisation abilities to show the
+It has rich visualisation abilities to show the
 available data extents, and can be used to browse the provenance of indexed data.
 
 See the `Digital Earth Australia Explorer`_ for an example deployment showing the power of this tool.
 
-.. _`Data Cube Explorer`: https://github.com/opendatacube/datacube-explorer
-.. _`Digital Earth Australia Explorer`: https://explorer.sandbox.dea.ga.gov.au
-
+(Depends on `odc-geo`_ and `datacube-core`_)
 
-Cube in a Box
--------------
+Indexing and Packaging Utilities
+++++++++++++++++++++++++++++++++
 
-`Cube in a Box`_ provides everything needed to get up and running quickly with Open Data Cube inside
-an Amazon Web Services Environment.
+DC Tools
+--------
 
-.. _`Cube in a Box`: https://github.com/opendatacube/cube-in-a-box
+`odc-apps-dc-tools`_ provides a collection of command line tools for indexing large data collections
+into an ODC database from various locations, including local file system, s3 buckets. Note that these tools
```

Comment:
Should "local file system, s3 buckets." be "local file system and S3 buckets"? (And capital S on S3?)